code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -189,6 +192,70 @@
 			SPAN_NOTICE("You pat [target] on the back to make [t_him] feel better!"), null, 4)
 	playsound(target, 'sound/weapons/thudswoosh.ogg', 25, 1, 5)
 
+/datum/species/proc/attempt_rock_paper_scissors(var/mob/living/carbon/human/H, var/mob/living/carbon/human/target)
+	if(!H.get_limb("r_hand") && !H.get_limb("l_hand"))
+		to_chat(H, SPAN_NOTICE("You have no hands!"))
+		return
+
+	if(!target.get_limb("r_hand") && !target.get_limb("l_hand"))
+		to_chat(H, SPAN_NOTICE("They have no hands!"))
+		return
+
+	//Responding to a raised hand
+	if(target.flags_emote & EMOTING_ROCK_PAPER_SCISSORS && do_after(H, 5, INTERRUPT_MOVED, EMOTE_ICON_ROCK_PAPER_SCISSORS))
+		if(!(target.flags_emote & EMOTING_ROCK_PAPER_SCISSORS)) //Additional check for if the target moved or was already high fived.
+			to_chat(H, SPAN_NOTICE("Too slow!"))
+			return
+		target.flags_emote &= ~EMOTING_ROCK_PAPER_SCISSORS
+		var/static/list/game_quips = list("Rock...", "Paper...", "Scissors...", "Shoot!")
+		for(var/quip in game_quips)
+			if(!H.Adjacent(target))
+				to_chat(list(H, target), SPAN_WARNING("You need to be adjacent to eachother to play!"))
+				return
+			to_chat(list(H, target), SPAN_NOTICE(quip))
+			sleep(5)
+		var/static/list/intent_to_play = list(
+			"[INTENT_HELP]" = "random",
+			"[INTENT_DISARM]" = "scissors",
+			"[INTENT_GRAB]" = "paper",
+			"[INTENT_HARM]" = "rock"
+		)
+		var/static/list/play_to_emote = list(
+			"rock" = EMOTE_ICON_ROCK,
+			"paper" = EMOTE_ICON_PAPER,
+			"scissors" = EMOTE_ICON_SCISSORS
+		)
+		var/protagonist_plays = intent_to_play["[H.a_intent]"] == "random" ? pick("rock", "paper", "scissors") : intent_to_play["[H.a_intent]"]
+		var/antagonist_plays = intent_to_play["[target.a_intent]"] == "random" ? pick("rock", "paper", "scissors") : intent_to_play["[target.a_intent]"]
+		var/winner_text = " It's a draw!"
+		if(protagonist_plays != antagonist_plays)
+			var/static/list/what_beats_what = list("rock" = "scissors", "scissors" = "paper", "paper" = "rock")
+			if(antagonist_plays == what_beats_what[protagonist_plays])
+				winner_text = " [H] wins!"
+			else
+				winner_text = " [target] wins!"
+		H.visible_message(SPAN_NOTICE("[H] plays <b>[protagonist_plays]</b>![winner_text]"), SPAN_NOTICE("You play <b>[protagonist_plays]</b>![winner_text]"), max_distance = 5)
+		target.visible_message(SPAN_NOTICE("[target] plays <b>[antagonist_plays]</b>![winner_text]"), SPAN_NOTICE("You play <b>[antagonist_plays]</b>![winner_text]"), max_distance = 5)
+		playsound(target, "clownstep", 35, TRUE)
```

Comment:
?

Reply:
funny sound that plays after a match, it was a placeholder until i tried it in testing and it was actually kinda fitting